command/agent_monitor.go

```diff
@@ -89,6 +89,28 @@ func (c *MonitorCommand) Run(args []string) int {
 		return 1
 	}
 
+	// Query the node info and lookup prefix
+	if len(nodeID) == 1 {
```

Comment:
should this check and the next be combined into len(nodeID) < 2?

Reply:
node-id could be blank if targeting a server so I think we need both

Reply:
👍